media/js/base/gtm/ie/gtm-snippet-ie-init.es6.js

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Note for reviewer: most of the code here is ported directly from https://github.com/mozilla/bedrock/blob/main/media/js/ie/gtm-snippet.js rather than being new.